### PR TITLE
fix lost messages

### PR DESCRIPTION
**Pull request by PolinaRodimova**

### CODE REVIEW

**Comment by PolinaRodimova** [comment]
more details: https://github.com/DataDog/java-dogstatsd-client/issues/59

**Comment by gzussa** [comment]
@tylerbenson @mar-kolya @palazzem Including you folks as reviewers here since this is Java

File: src/test/java/com/timgroup/statsd/NonBlockingStatsDClientTest.java [hunk 1]
```diff
@@ -594,4 +597,60 @@ public void clear() {
         assertThat(server.messagesReceived(), contains("my.prefix.myset:myuserid|s|#baz,foo:bar"));
 
     }
+
+    @Test(timeout = 5000L) public void
+    shutdown_test() throws Exception {
```
[the remark this answers is not in the document]
Comment (tylerbenson) [reply src/test/java/com/timgroup/statsd/NonBlockingStatsDClientTest.java]: Oops, I'm obviously not familiar with this code. I didn't realize you were being consistent.  Sorry about that.  You can change it back if you want, or leave it as is.

[the remark this answers is not in the document]
Comment (PolinaRodimova) [reply src/test/java/com/timgroup/statsd/NonBlockingStatsDClientTest.java]: I prefer to keep these changes, previous version looked strange for me:)

**Comment by gzussa** [comment]
Actually never mind since there is an issue linked to it

**Pull request merged**